src/main/java/com/buuz135/industrial/tile/agriculture/AnimalResourceHarvesterTile.java

```diff
@@ -92,7 +95,12 @@ public float work() {
                 }
                 hasWorked = true;
             }
-            if (tank.getFluidAmount() <= 7000) {
+            //Check if the animal is a Fluid Cow from moofluids. If the cow has a different
+            //fluid than what is in the tank, skip it
+            if (tank.getFluidAmount() <= 7000 && 
+            		!(Loader.isModLoaded("moofluids") && tank.getFluidAmount() > 0 && 
+            		living instanceof EntityFluidCow &&
+            			((EntityFluidCow)living).getEntityFluid() != tank.getFluid().getFluid())) {
```

Comment:
This will crash if the mod is missing

Reply:
I already tested this. I loaded up a world without MooFluids, and it didn't crash. This check makes it safe: `Loader.isModLoaded("moofluids")`

Reply:
You are importing MooFluids classes, have you tried placing a Rancher down to check if it crashed?

Reply:
Yup. Here is a GIF showing my test:  https://imgur.com/a/3E1BHY8

Reply:
As far as I know, the MooFluid classes aren't being accessed because the `if` statement fails before it gets to them. I'm pretty sure this is correct usage of `isModLoaded`. But, I could be wrong.